### PR TITLE
manifesto

### PR DESCRIPTION
**Pull request by frankiebee**
this is a proposal on how to be

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,8 +107,7 @@
     "uuid": "^9.0.1"
   },
   "lint-staged": {
-    "*.ts": "eslint --fix",
-    "*.{js,ts,json,md}": "prettier --write"
```
Comment (mixmix) [inline package.json:113]: different PR for this plz

**Review by mixmix — COMMENTED** [review]
Great!

i think we need to say what we mean by solarpunk more - feminist, environmentalist, collectivist, mongrel, intersectional

come at me on git fetch. I love that command

**Review by mixmix — APPROVED** [review]

**Comment by mixmix** [comment]
@frankiebee tweak + merge!

**Review by rh0delta — APPROVED** [review]

**Comment by mixmix** [comment]
HERE WE GO!

**Pull request merged**